Octokit.Tests.Integration/Clients/PullRequestsClientTests.cs

```diff
@@ -653,6 +653,25 @@ public async Task CanClose()
         Assert.Equal(pullRequest.Body, result.Body);
     }
 
+    [IntegrationTest]
+    public async Task CanLockAndUnlock()
+    {
+        await CreateTheWorld();
+
+        var newPullRequest = new NewPullRequest("a pull request to lock", branchName, "main");
+        var pullRequest = await _fixture.Create(Helper.UserName, _context.RepositoryName, newPullRequest);
+
+        await _fixture.LockUnlock.Lock(Helper.UserName, _context.RepositoryName, pullRequest.Number, LockReason.OffTopic);
+        var retrived = await _fixture.Get(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+
+        Assert.Equal(retrived.ActiveLockReason, LockReason.OffTopic);
+
+        await _fixture.LockUnlock.Unlock(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+        var unlocked = await _fixture.Get(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+
+        Assert.Equal(unlocked.ActiveLockReason, null);
```

Comment:
Please could you address the warnings just below here?

Reply:
Thanks for catching that. Updated the PR with Assert.Null.